app/views/organisations/_related_people.html.erb

```diff
@@ -1,5 +1,12 @@
 <% if people.any? %>
-  <section class="brand--<%= brand %> brand__border-color organisation__brand-border-top organisation__margin-bottom">
+  <%
+    ga4_image_card_json = {
+      "event_name": "navigation",
+      "type": "image card",
+      "section": title
+    }.to_json
+  %>
+  <section class="brand--<%= brand %> brand__border-color organisation__brand-border-top organisation__margin-bottom" data-people-id="<%= title.downcase.split(' ').join('-')%>">
```

Comment:
I added `data-people-id` for testing purposes, as these elements had no distinguishable CSS selector.